agents/ocf/HealthIOWait.in

```diff
@@ -60,6 +60,15 @@ the #health-iowait will go red if the %iowait of the CPU get higher than 15%.
 <content type="integer" default="15"/>
 </parameter>
 
+<parameter name="dampening" reloadable="1">
```

Comment:
In order for this to be truly reloadable, the agent also has to define the reload-agent action, even if it does nothing. However it appears to be a mistake that this was made reloadable in other agents ... attrd_updater -U will only set the dampening if the attribute doesn't exist, so reloading wouldn't actually change it. It really does need a full restart to delete the attribute and re-add it with the new dampening. So, just drop reloadable here.

Reply:
OK, HealthCPU uses `-B` instead of `-U`, so dampening would work as a reloadable parameter there (`-B` updates both the value and the dampening). HealthSMART uses `-U`, so it wouldn't work.

You can either use `-B`, or drop `reloadable`.

Reply:
Thanks for sharing. I haven't tried this function before, and I plan to change the -U of all agents to -B